Source/RapyutaSimulationPlugins/RapyutaSimulationPlugins.Build.cs

```diff
@@ -113,5 +113,10 @@ public RapyutaSimulationPlugins(ReadOnlyTargetRules Target) : base(Target)
         {
             AddLib(Target, SdformatPath, "libsdformat12.so", "libsdformat12.so.12");
         }
+
+        // two_point_interpolation
+        string TwoPointsInterpolationPath = GetLibPath("two_points_interpolation_cpp");
+        PublicIncludePaths.Add(TwoPointsInterpolationPath);
```

Comment:
If the method is not so big a lib, could we just have it added directly into RRMathUtils?

Reply:
it is just one file header lib but want to keep in thirdparty but I can avoid submodule and directly add file.

Reply:
Yep, it'd depend on how you might expect the lib to grow, so up to you then. Inline adding the src to our UE plugin would have to come with a catch of possible future certain dependencies & coding convention, which are not always easily assured.

Reply:
Yep, it'd depend on how you might expect the lib to grow, so up to you then. Inline adding the src to our UE plugin could come with a catch of possible future certain dependencies & coding convention, which are not always easily assured.